tests/test_validate_metadata.py

```diff
@@ -250,10 +253,63 @@ def test_valid_array_content(self):
         )
         self.teardown_metadata(metadata)
 
+    def test_external_metadata_convention(self):
+        """
+        check that extermal_metadata_convention has been staged to gs://broad-singlecellportal-public
+        """
+        # obtain local and external convention info from IngestPipeline object
+        ingest_object = IngestPipeline(
+            ObjectId('aaaaaaaaaaaaaaaaaaaaaaaa'), ObjectId('aaaaaaaaaaaaaaaaaaaaaaaa')
```

Comment:
Non-blocking: you can simply call `ObjectId()` without passing any data to generate a new ID.

Reply:
Thanks, I didn't know that :)